brainscore_vision/model_helpers/brain_transformation/temporal.py

```diff
@@ -30,17 +128,23 @@ def visual_degrees(self) -> int:
 
     def look_at(self, stimuli, number_of_trials=1):
         responses = self._layer_model.look_at(stimuli, number_of_trials=number_of_trials)
-        time_responses = []
-        self._logger.debug(f'Repeating single assembly across time bins {self._time_bins}')
-        for time_bin in self._time_bins:
-            time_bin = time_bin if not isinstance(time_bin, np.ndarray) else time_bin.tolist()
-            time_bin_start, time_bin_end = time_bin
-            bin_responses = responses.expand_dims('time_bin_start').expand_dims('time_bin_end')
-            bin_responses['time_bin_start'] = [time_bin_start]
-            bin_responses['time_bin_end'] = [time_bin_end]
-            bin_responses = bin_responses.stack(time_bin=['time_bin_start', 'time_bin_end'])
-            time_responses.append(bin_responses)
-        responses = merge_data_arrays(time_responses)
+        if "time_bin" not in responses.dims:
+            # if the model does not support temporal output, assume the output is the same across time
+            time_responses = []
+            self._logger.debug(f'Repeating single assembly across time bins {self._time_bins}')
+            for time_bin in self._time_bins:
+                time_bin = time_bin if not isinstance(time_bin, np.ndarray) else time_bin.tolist()
+                time_bin_start, time_bin_end = time_bin
+                bin_responses = responses.expand_dims('time_bin_start').expand_dims('time_bin_end')
+                bin_responses['time_bin_start'] = [time_bin_start]
+                bin_responses['time_bin_end'] = [time_bin_end]
+                bin_responses = bin_responses.stack(time_bin=['time_bin_start', 'time_bin_end'])
+                time_responses.append(bin_responses)
+            responses = merge_data_arrays(time_responses)
```

Comment:
suggest extracting into its own method to keep the `look_at` method lean (like you did with `assembly_time_align`)